### PR TITLE
Fixes #4289 - use same div class in all notice bars

### LINKED ISSUE #4289: global_moderation_notice - use same div class like other notice bars
pm_alert should be changed to red_alert to keep consistency

### PR DESCRIPTION
**Pull request by Eldenroot**
Resolves #4289

### CODE REVIEW

File: install/resources/mybb_theme.xml [hunk 1]
```diff
@@ -4206,7 +4206,7 @@ $(".forumjump").on('change', function() {
 </div>]]></template>
 		<template name="global_remote_avatar_notice" version="1809"><![CDATA[<div class="red_alert"><a href="{$mybb->settings['bburl']}/usercp.php?action=avatar">{$lang->remote_avatar_disabled_default_avatar}</a></div>]]></template>
 		<template name="global_unreadreports" version="1820"><![CDATA[<a href="{$mybb->settings['bburl']}/modcp.php?action=reports">{$lang->unread_reports}</a>]]></template>
-		<template name="global_moderation_notice" version="1800"><![CDATA[<div class="pm_alert">{$moderation_text}</div>]]></template>
```
Comment (Sama34) [inline install/resources/mybb_theme.xml:4209]: Perhaps the version should now be `1827`? I don't know if these are updated for closed PRs ..

**Review by Sama34 — APPROVED** [review]

**Comment by Ben-MyBB** [comment]
Probably best to update the version I think then we can merge

File: install/resources/mybb_theme.xml (hunk 1 above)
Comment (Eldenroot) [reply install/resources/mybb_theme.xml:4209]: Done!

**Review by Eldenroot — COMMENTED** [review]
Done!

**Pull request merged**